### PR TITLE
link: fix old/bad Get Started links

### PR DESCRIPTION
**Pull request by jorgeorpinel**
This is self-approved maintenance work, but need a formal approval to merge.

### CODE REVIEW

File: content/docs/command-reference/get.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ file or directory from. It also has the `--out` option to specify the location
 to place the target data within the workspace. Combining these two options
 allows us to do something we can't achieve with the regular `git checkout` +
 `dvc checkout` process – see for example the
-[Get Older Data Version](/doc/tutorials/get-started/data-versioning#navigate-versions)
+[Get Older Data Version](/doc/start/data-versioning#switching-between-versions)
```
Comment (jorgeorpinel) [inline content/docs/command-reference/get.md:154]: + anchor fix

File: content/docs/command-reference/metrics/index.md [hunk 2]
```diff
@@ -102,9 +102,9 @@ to compare and pick the best performing experiment.
 
 ## Examples
 
-> This example is based on our
-> [Get Started](/doc/start/metrics-parameters-plots#collecting-metrics), where
-> you can find the actual source code.
+> This example is based on the `evaluate` stage of our
+> [Get Started](/doc/start/metrics-parameters-plots), where you can find the
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/index.md:106]: Remove anchor

File: content/docs/command-reference/metrics/show.md [hunk 3]
```diff
@@ -63,8 +63,8 @@ compares them with a previous version.
 ## Examples
 
 > This example is based on the `evaluate` stage of our
-> [Get Started](/doc/start/metrics-parameters-plots#collecting-metrics), where
-> you can find the actual source code.
+> [Get Started](/doc/start/metrics-parameters-plots), where you can find the
```
Comment (jorgeorpinel) [inline content/docs/command-reference/metrics/show.md:66]: "

File: content/docs/start/data-pipelines.md [hunk 4]
```diff
@@ -86,7 +86,7 @@ The command options used above mean the following:
 
 - `-p prepare.seed,prepare.split` defines special types of dependencies -
   [parameters](/doc/command-reference/params). We'll get to them later in the
-  [Metrics, parameters, and plots](/doc/start/metrics-parameters-plots) section,
+  [Metrics, Parameters, and Plots](/doc/start/metrics-parameters-plots) page,
```
Comment (jorgeorpinel) [inline content/docs/start/data-pipelines.md:89]: Copy edit

File: content/docs/start/data-pipelines.md [hunk 5]
```diff
@@ -143,8 +143,8 @@ stages:
 There's no need to use `dvc add` for DVC to track stage outputs (`data/prepared`
 in this case); `dvc run` already took care of this. You only need to run
 `dvc push` if you want to save them to
-[remote storage](/doc/tutorials/get-started/data-versioning#storing-and-sharing),
-(usually along with `git commit` to version `dvc.yaml` itself).
+[remote storage](/doc/start/data-versioning#storing-and-sharing), (usually along
```
Comment (jorgeorpinel) [inline content/docs/start/data-pipelines.md:146]: + remove anchor

File: content/docs/start/metrics-parameters-plots.md [hunk 6]
```diff
@@ -16,7 +16,7 @@ Read on to see how it's done!
 
 First, let's see what is the mechanism to capture values for these ML
 attributes. Let's add a final evaluation stage to our
-[pipeline](/doc/start/data-pipelines#dependency-graphs-dags):
+[pipeline](/doc/start/data-pipelines):
```
Comment (jorgeorpinel) [inline content/docs/start/metrics-parameters-plots.md:19]: Rm anchor

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md [hunk 7]
```diff
@@ -345,15 +345,16 @@ very convenient having to remember to do so every time the dataset changes.
 Here's where the [pipelines](/doc/command-reference/dag) feature of DVC comes in
 handy. We touched on it briefly when we described `dvc run` and `dvc repro`. The
 next step would be splitting the script into two parts and utilizing pipelines.
-See [Data Pipelines](/doc/start/data-pipelines) to get hands-on experience with
-pipelines, and try to apply it here. Don't hesitate to join our
+See [Get Started: Data Pipelines](/doc/start/data-pipelines) to get hands-on
```
Comment (jorgeorpinel) [inline content/docs/use-cases/versioning-data-and-model-files/tutorial.md:348]: Complete title

File: content/docs/user-guide/project-structure/pipelines-files.md [hunk 8]
```diff
@@ -4,7 +4,7 @@ You can construct data science or machine learning pipelines by defining
 individual [stages](/doc/command-reference/run) in one or more `dvc.yaml` files
 (or _pipelines files_). Stages form a pipeline when they connect with each other
 (forming a _dependency graph_, see `dvc dag`). Refer to
-[Data Pipelines](/doc/start/data-pipelines).
+[Get Started: Data Pipelines](/doc/start/data-pipelines).
```
Comment (jorgeorpinel) [inline content/docs/user-guide/project-structure/pipelines-files.md:7]: "

File: content/docs/use-cases/versioning-data-and-model-files/tutorial.md [hunk 9]
```diff
@@ -345,15 +345,16 @@ very convenient having to remember to do so every time the dataset changes.
...
+experience with pipelines, and try to apply it here. Don't hesitate to join our
 [community](/chat) and ask any questions!
 
 Another detail we only brushed upon here is the way we captured the
 `metrics.csv` metrics file with the `-M` option of `dvc run`. Marking this
 <abbr>output</abbr> as a metric enables us to compare its values across Git tags
 or branches (for example, representing different experiments). See
 `dvc metrics`,
-[Comparing Changes](/doc/start/metrics-parameters-plots#comparing-changes), and
+[Comparing Changes](/doc/start/metrics-parameters-plots#comparing-iterations),
```
Comment (jorgeorpinel) [inline content/docs/use-cases/versioning-data-and-model-files/tutorial.md:357]: Anchor fix

File: content/docs/user-guide/large-dataset-optimization.md [hunk 10]
```diff
@@ -4,11 +4,11 @@ In order to track the data files and directories added with `dvc add`,
 `dvc repro`, etc. DVC moves all these files to the project's <abbr>cache</abbr>.
 
 However, the versions of the tracked files that
-[match the current code](/doc/tutorials/get-started/data-pipelines) are also
-needed in the <abbr>workspace</abbr>, so a subset of the cached files can be
-kept in the working directory (using `dvc checkout`). Does this mean that some
-files will be duplicated between the workspace and the cache? **That would not
-be efficient!** Especially with large files (several Gigabytes or larger).
+[match the current code](/doc/user-guide/project-structure) are also needed in
```
Comment (jorgeorpinel) [inline content/docs/user-guide/large-dataset-optimization.md:7]: Linked to somewhere else

**Review by jorgeorpinel — COMMENTED** [review]
Everything here does /doc/tutorials/get-started -> /doc/start except the ones below (some related to @dberenbaum's latest Get Started update):

**Review by shcheklein — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
A link is failing. Typo I think... ⌛

**Pull request merged**